tests/bluetooth/common/testlib/include/bs_helpers/flags.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/atomic.h>
+
+#define DECLARE_FLAG(flag) extern atomic_t flag
```

Comment:
Ditto as for https://github.com/zephyrproject-rtos/zephyr/pull/71053/files#r1551037606 we should document these

Reply:
I feel stupid writing the docs, the macros are litterally faster to read than the doxygen comment

Reply:
> I feel stupid writing the docs, the macros are litterally faster to read than the doxygen comment

That's because you're smart :D